migration/20180310-add-loan-hold-integration-client.sql

```diff
@@ -0,0 +1,33 @@
+DO $$
```

Comment:
Please bump the date on this migration script; I landed two scripts yesterday and don't want them to run out of order.